CMSgov/mint-app
# [EASI-4453] Tooltip for PBG Filtered View

# EASI-4453

## Description
- Add the tooltips to the filtered view (see list in JIRA ticket) 
- Extended the translation with `groupLabel` & `groupLabelTooltip`





## How to test this change

1. Go through a model plan and answer questions listed in the JIRA ticket
2. Filter read view to `PBG`
3. Verify that tooltip shows up and matches figma

- 

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
